client-service-worker/gulpfile.js

```diff
@@ -0,0 +1,61 @@
+const gulp = require('gulp');
+const gap = require('gulp-append-prepend');
+const del = require('del');
+const merge2 = require('merge2');
+
+const {NODE_ENV} = process.env;
+
+const isProd = NODE_ENV === 'production';
+const isTest = NODE_ENV === 'test';
+
+const JSDELIVR_CDN_DOMAIN = process.env.JSDELIVR_CDN_DOMAIN ? process.env.JSDELIVR_CDN_DOMAIN.trim() : 'cdn.jsdelivr.net';
```

Comment:
Is there another reason for `trim()`? I thought it's quite hard to unintentionally add white spaces to env variable, it actually takes some effort.

I think this is sufficient:
```js
const JSDELIVR_CDN_DOMAIN = process.env.JSDELIVR_CDN_DOMAIN || 'cdn.jsdelivr.net';
```

Reply:
Yes there is usually no problem, I just had some during my tests under Windows (command line). But actually, in real case, there won't be these problems.